### PR TITLE
ShipIt from Anywhere blog post

### PR DESCRIPTION
**Pull request by cchesser**
Planned for publishing this week (est. Dec 3rd)

### CODE REVIEW

File: content/post/shipit-from-anywhere-lessons-learned/index.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+Moving this event to a virtual format has created a lot of challenges! During a typical in-person ShipIt event, all participating teams would gather in one room to collaborate on their projects. Participating teams would be provided with awesome swag, a fully loaded snack bar and an evening movie to give a fun atmosphere for those who enjoyed to be attending in the evening (check out our recap video for what it was like on our campus). Another very anticipated part of the event is the live demo portion where each participating team shares what they created in the last 24 hours. At the end of the presentations, the judges and people-choice winners get announced in our assembly, where each team then gets to pick from a line-up of prizes, with the 1st place team receiving the traveling trophy: the golden keyboard. It was a fun moment to wrap up the event and capture photos of all the teams.
```
Comment (WhitneyRoark) [inline content/post/shipit-from-anywhere-lessons-learned/index.md:11]: The recap video needs to be linked on "check out our recap video". 
See the link for it here : https://www.youtube.com/user/CernerEng/search?query=shipit

File: content/post/shipit-from-anywhere-lessons-learned/index.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
+---
+title: "ShipIt from Anywhere: Our Lessons on Going Virtual"
+authors: ["Whitney Roark"]
+date: 2020-12-03
+tags: [culture, engineering, hackathon, shipit]
+thumbnail: "shipit-shirts.png"
+---
+
+ShipIt Days are Cerner Engineering's 24-hour hackathons where associates are empowered to brainstorm, plan, build, and present a project of their choice. This quarterly event is something associates enjoy by taking a break from their normal routine and focusing on a problem that they seek to achieve within a short time period (one day). The enjoyment of these projects often come from just working on something different, working with other colleagues, and just applying their passions to an area they have been desiring to solve. With the pandemic this year, we had to rethink how can we continue to achieve this same associate enjoyment and fulfillment while only being in a virtual format.
```
Comment (WhitneyRoark) [inline content/post/shipit-from-anywhere-lessons-learned/index.md:9]: Can we remove the word "just" before "applying their passions" it sounds a bit odd.

**Review by WhitneyRoark — COMMENTED** [review]
Left some comments with potential changes

File: content/post/shipit-from-anywhere-lessons-learned/index.md (hunk 2 above)
Comment (cchesser) [reply content/post/shipit-from-anywhere-lessons-learned/index.md]: Applied changes: 5ad6e07

File: content/post/shipit-from-anywhere-lessons-learned/index.md (hunk 1 above)
Comment (cchesser) [reply content/post/shipit-from-anywhere-lessons-learned/index.md]: Applied changes: 5ad6e07

**Review by WhitneyRoark — APPROVED** [review]

**Pull request merged**